### ISSUE TITLE
When sending of a post with attachment to a community failed, i wanted to delete, but the post remains.

### ISSUE THREAD

**Issue by slrslr** [oxen-io/session-desktop#2704]
**Describe the bug**
I have tried to post a sentence with an .webm(likely does not matter) attachment to a "General chat", "Public group", "Maddisyns Chat" communities and it **failed (red circle icon), resend fails** (my bet is that these communities disabled attachments/upload permission), my test community accepts same post with attachment.

At this point where post sending failed, i want to prevent this attachment potentially taking up space in my Session app data folder and also cleanup the timeline in that community from that post. But only option to **Delete for all does not remove the post**. Can you reproduce this?

**Other information (please complete the following information):**

* Device: PC
* OS: Debian + KDE
* Session Version or Git commit hash: 1.10.7

**Comment by ianmacd** [comment]
> Can you reproduce this?

Absolutely. I see this behaviour regularly in my own test group. It's impossible to remove failed messages.

**Comment by KeeJef** [comment]
Yeah this should be fixed, Good first issue to add the normal options to delete to failed to send messages

**Comment by slrslr** [comment]
Last version/s this issue no longer happen in case i select "Delete just for me".
I suppose it should pre-select this option (to delete just for me) in case of a failed message or display only this option. But maybe that would be different issue..
